utils/helpers.py

```diff
@@ -106,7 +106,7 @@ def write_record_skipped(record, file_path):
     :return: None
     """
     filename = "/skipped.csv"
-    fieldnames = ('Count', 'Make', 'Type', 'Asset Code', 'Category',
+    fieldnames = ('count', 'Make', 'Type', 'Asset Code', 'Category',
```

Comment:
All the field names are in title case. What's the reasoning behind changing the "count" field name to lowercase?

Reply:
It was conflicting with the field names in the dictionary.

Reply:
What dictionary @andela-Taiwo?